integration-tests/pages/checkAnswersPage.js

```diff
@@ -1,12 +1,20 @@
 const page = require('./page')
 
 const clickSubmit = () => cy.get('[data-submit]').click()
+const incidentDateFormat = new RegExp(Cypress.moment().format('DD/MM/YYYY'))
+const incidentTimeFormat = new RegExp(Cypress.moment().format('HH:mm'))
```

Comment:
won't this fail if the start of the test and end of the test span the minute boundary